client/src/shared/components/Text-field/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+interface Props {
```

Comment:
- this interface could be usable in the future development. so please export it.
```suggestion
export interface Props {
```